web/packages/core/src/shadow-template.ts

```diff
@@ -227,3 +227,37 @@ ruffleShadowTemplate.innerHTML = `
 
     <ul id="context-menu"></ul>
 `;
+
+export const preloaderTemplate = document.createElement("template");
+preloaderTemplate.innerHTML = `
+    <div id="preloader">
+        <style>
+            #preloader {
+                display: flex;
+                flex-direction: column;
+                background: var(--ruffle-blue);
```

Comment:
background: var(--preloader-background, var(--ruffle-blue));

That way, people who want to use a different background color can add this to the main site's CSS:

```
:root {
    --preloader-background: none; // or different color
}
```